setup.py

```diff
@@ -32,7 +32,8 @@
                   'isort',
                   'pytest-xdist',
                   'pytest-cov',
-                  'coverage'
+                  'coverage',
+                  'typeguard==2.7.1'
```

Comment:
why are we pinning the version of this one?
Is it stable?

Reply:
It's stable, I'm just a bit too careful sometimes haha